src/__tests__/CompareResults/beta/ResultsTable.test.tsx

```diff
@@ -19,4 +19,10 @@ describe('Results Table', () => {
     expect(screen.getByTestId('results-table')).toBeInTheDocument();
     expect(document.body).toMatchSnapshot();
   });  
+
+  it('Display message for not finding results', () => {
+    renderWithRouter(<ResultsTable themeMode={themeMode} />);
+
+    expect(screen.getByTestId('no-results-found-msg')).toBeInTheDocument();
```

Comment:
```suggestion
    expect(screen.getByText(/No results found/)).toBeInTheDocument();
```
And remove the useless test-id.

In general we try to avoid using test-ids and rely instead on what the user sees or can interact with.